### PR TITLE
Adding compatibility measures to address the deprecation of names in …

### PR DESCRIPTION
**Pull request by pkofod**
…favour of fieldnames. Also deleted trailing whitespace, and cleaned up indentation.

No functionality should have changed! :)

### CODE REVIEW

File: src/dataframe.jl [hunk 1]
```diff
@@ -6,153 +6,153 @@
...
-    
+
     #Empty df as container for results
     resultdf = DataFrame()
-    
+
     #Get array of field names as symbols from composite type
-    cols = names(TWEETS) 
-    
+    @compat cols = fieldnames(TWEETS)
```
Comment (randyzwitch) [inline src/dataframe.jl]: Is this literally the only code change? Everything else is spacing?

Comment (pkofod) [inline src/dataframe.jl]: That and line 5 in REQUIRE and line 9 in Twitter.jl just to make sure @compat is in the namespace. Everything else is cosmetics :)

Comment (randyzwitch) [inline src/dataframe.jl]: The space changes are fine, it was just hard for me to tell what was different across the 21 files.

Comment (pkofod) [inline src/dataframe.jl]: Yes, I'm aware. I should probably have made two different commits.

**Pull request merged**